packages/plugins/package.json

```diff
@@ -58,7 +58,7 @@
     "winston": "^3.8.2"
   },
   "peerDependencies": {
-    "@glint/core": "^1.1.0",
+    "@glint/core": "^1.2.1",
```

Comment:
Generally speaking bumping the minimum of a package in peer dependencies is a breaking change. However, for this specific case I searched GitHub (OSS) and internal repos for reference to `@rehearsal/plugins` (outside of lockfiles), and only found `@rehearsal/migrate` (which is also updated in this PR). So this is practically speaking going to be fine.